src/AutoRest.CSharp/Mgmt/Generation/MgmtClientBaseWriter.cs

```diff
@@ -848,7 +863,7 @@ protected virtual void WriteLROResponse(string diagnosticsVariableName, string p
         }
         #endregion
 
-        protected void WriteArguments(CodeWriter writer, IEnumerable<ParameterMapping> mapping, bool passNullForOptionalParameters = false)
+        protected void WriteArguments(CodeWriter writer, IEnumerable<ParameterMapping> mapping, bool passNullForOptionalParameters = false, bool isPropertyBagMethod = false)
```

Comment:
Can we remove this parameter and use [parameter.Parameter.IsPropertyBag](https://github.com/Azure/autorest.csharp/pull/2862/files#diff-1d8080793123f1f633e4fcb80852199dacf38a51b2fd96f50e256756507f0429R24) instead?

Reply:
Removed